nixos/modules/services/networking/rpki-client.nix

```diff
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.rpki-client;
+  tals = [
+    "${cfg.package}/etc/rpki/afrinic.tal" "${cfg.package}/etc/rpki/apnic.tal"
+    "${cfg.package}/etc/rpki/lacnic.tal" "${cfg.package}/etc/rpki/ripe.tal" ]
+    ++ optional cfg.accept-arin-rpa "${pkgs.rpki-arin-tal}/arin.tal"
```

Comment:
```suggestion
    ++ optional cfg.accept-arin-rpa "${cfg.package}/arin.tal"
```

Reply:
no, the purpose of this switch is to use pkgs.rpki-arin-tal